### ISSUE TITLE
Can anybody reproduce the results in the paper

### ISSUE THREAD

**Issue by niyunsheng** [yyk-wew/F3Net#13]
original下的AUC比其他mode下的都高，是不是可以认为这篇文章复现不了？
我自己用这个代码跑的也是和readme中类似的结果

**Comment by M-SunRise** [comment]
Yes, I got the similar results to yours. Although I have added the mix block into the net, it works weakly.

**Comment by yyk-wew** [comment]
I've tried the mix block, too. But it runs not very well.

> original下的AUC比其他mode下的都高，是不是可以认为这篇文章复现不了？ 我自己用这个代码跑的也是和readme中类似的结果

Yes. For now, this is what I've got.

However, on the other hand, I have to remind you that this repo is just a personal version of F3Net. Too many implementation details missing in the paper (especially for mix block)  makes the reproduction more difficult. Thus I can't promise that this is the "right" version of what the paper proposed.
